src/locale-data.js

```diff
@@ -6,6 +6,7 @@ import am from './locale-data/am';
 import ar from './locale-data/ar';
```

Comment:
Does `an` need to be added here?

Reply:
Usually it would, but Aragonese (an) was never implemented for react-intl locale-data, so there is no `an.json` file in `src/locale-data`. Our work around for languages that don't have locale-data is to set up what's called a custom locale in supported-locales, and use a `parent` locale that does exist. For Aragonese I used `es` as the parent locale since it's a part of Spain.